### DISCUSSION TITLE
commits still showing 0

### DISCUSSION THREAD

**Discussion by MitkoVtori** [anuraghazra/github-readme-stats#2308] [Q&A] [answered]
I still recieve 0 commits when I use `include_all_commits=true`
When it shows 0, I just had to add some random junk infront of the `include_all_commits` like, for example:
`include_all_commits=true&include_something_invalid=true`, this worked for 1-2 days and then it again shows 0, so I had to remove it. Now it stays for 1 hour so I have to change it every hour. Why is this happening? How can I fix it?
![Screenshot 2022-11-30 184434](https://user-images.githubusercontent.com/112943652/204858141-4b879f8b-b674-42dd-b4e8-6cb64fc8b330.png)
### I also had vercel and followed the steps for instaling it like you said.

**Comment by rickstaa** [comment, marked as answer]
@MitkoVtori Thanks for your question. This is a known problem. When the GitHub Rest API fails, we use 0 as a fallback. Currently, our codebase saves this fallback value in the cache. I created https://github.com/anuraghazra/github-readme-stats/pull/2177 to address this. However, @anuraghazra needs to review this PR before merging it.

![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=MitkoVtori&count_private=true&include_all_commits=true&random=23423ads)

```md
![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=MitkoVtori&count_private=true&include_all_commits=true&random=23423ads)
```

**Reply by MitkoVtori** [reply]
Thank you for your hard work :)
